pulpcore/pulpcore/app/models/repository.py

```diff
@@ -395,6 +407,9 @@ def remove_content(self, content):
         if self.complete:
             raise ResourceImmutableError(self)
 
+        if not self.content_exists(content):
```

Comment:
Do we really want to raise an exception here?  Perhaps remove_content() should be idempotent.